devlib/utils/types.py

```diff
@@ -68,6 +68,15 @@ def numeric(value):
     """
     if isinstance(value, int):
         return value
+
+    if isinstance(value, basestring):
+        value = value.strip()
+        if value.endswith('%'):
+            try:
+                return float(value.rstrip('%')) / 100
```

Comment:
Should we do that in a more generic way ? We could do something like
```python
corrector = 1.0
if value.endswith('%'):
    value = value.rstrip('%')
    corrector = 0.01
# [ ... rest of the function ... ]
return fvalue*corrector
```
I think the code will be easier to read and it can be extended with no effort to account for other corner cases (_pre-mille_ for example: ‰)